### PR TITLE
[CHM 329] add radio component

### PR DESCRIPTION
**Pull request by liyuanmontreal**
https://github.com/scbd/scbd-common/assets/65098066/a3514498-b9de-49f1-bdb3-8121548fd2f1

### CODE REVIEW

**Review by blaisefonseca — COMMENTED** [review]
Can you add an attribute call `inline` and add class `form-check-inline` when the attribute is added. Do it for both radio and checkbox 
From bootstrap 

image

**Comment by liyuanmontreal** [comment]
add inline style

https://github.com/scbd/scbd-common/assets/65098066/87e4f136-a71e-4eda-9fad-7a278379030c

demo-radio2

**Review by blaisefonseca — APPROVED** [review]

File: src/components/inputs/radio.vue [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<template>
+    <div class="scbd-common radio  form-check form-check-inline">
```
Comment (blaisefonseca) [inline src/components/inputs/radio.vue]: only add inline class when the attribute `inline` exists 

`:class="{'form-check-inline' : $attrs.inline}"`

**Comment by liyuanmontreal** [comment]
[only show inline style when set inline = "true"
image

**Review by blaisefonseca — APPROVED** [review]

**Pull request merged**